### PR TITLE
ci: add iOS build and test action

### PR DESCRIPTION
**Pull request by tomaszczart**
This PR introduces building and running unit tests for iOS app by the CI. Here is [Apple documentation](https://developer.apple.com/library/archive/technotes/tn2339/_index.html#//apple_ref/doc/uid/DTS40014588-CH1-UNIT) describing used command in detail.

### CODE REVIEW

**Review by tomekdz — APPROVED** [review]

**Review by zofnow — APPROVED** [review]
I was wondering previously why there's no action to build the iOS app next to the one for tests, but I see the updated description now, so nvm :P

File: .github/workflows/readme.yml [hunk 1]
```diff
@@ -35,13 +35,12 @@ jobs:
     needs: commitlint
     runs-on: macos-latest
     steps:
+      - uses: actions/checkout@v2
```
Comment (wzieba) [inline .github/workflows/readme.yml:38]: 👀 🥇 !

File: .github/workflows/readme.yml [hunk 2]
```diff
@@ -35,13 +35,12 @@ jobs:
...
         with:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - uses: actions/checkout@v2
       - name: build and test
-        run: ./gradlew buildDebug
-
+        run: xcodebuild test -project iosApp/iosApp.xcodeproj -scheme iosApp -destination "platform=iOS Simulator,name=iPhone 11 Pro"
```
Comment (wzieba) [inline .github/workflows/readme.yml:46]: Can we also start ios tests? I think it'll be 
```
./gradlew iosX64Test
```

image

**Review by wzieba — CHANGES_REQUESTED** [review]

File: .github/workflows/readme.yml (hunk 2 above)
Comment (tomaszczart) [reply .github/workflows/readme.yml:48]: @wzieba Sure, added.

**Review by wzieba — APPROVED** [review]

**Pull request merged**